include/grpc/impl/codegen/grpc_types.h

```diff
@@ -142,6 +142,8 @@ typedef struct {
 /** Secondary user agent: goes at the end of the user-agent metadata
     sent on each request */
 #define GRPC_ARG_SECONDARY_USER_AGENT_STRING "grpc.secondary_user_agent"
+/** The maximum time between subsequent connection attempts, in ms */
```

Comment:
Any reason not to make this just seconds?
I can't imagine a use case where we'd want it to be subsecond, so allowing it seems like allowing shooting oneself in the foot.